kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build-model-image.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# 
+#  Summary:
+#
+#    This script builds a model image using the WebLogic Image Tool. The
+#    tool pulls a base image if there isn't already a local base image.
+#    This script, by default, builds the model image with model files from
+#    './stage-model-image.sh' plus tooling downloaded by './stage-tooling.sh.'.
+#
+#  Assumptions:
+#
+#    - The WebLogic Image Tool zip is 'WORKDIR/weblogic-image-tool.zip' and
+#      the WebLogic Deploy Tool zip is 'WORKDIR/weblogic-deploy-tooling.zip'
+#      (see './stage-tooling.sh').
+#
+#    - Model files have been staged in the MODEL_DIR directory
+#      (by './stage-model-image.sh' or some custom process).
+#
+#  Optional environment variables:
+#
+#    WORKDIR
+#      Working directory for the sample with at least 10GB of space.
+#      Defaults to '/tmp/$USER/model-in-image-sample-work-dir'.
+#
+#    MODEL_DIR:
+#      Location of the model .zip, .properties, and .yaml files
+#      that will be copied to the model image.  Default is:
+#        'WORKDIR/model/image--$(basename $MODEL_IMAGE_NAME):$MODEL_IMAGE_TAG'
+#      which is usually populated by the './stage-model-image.sh' script.
+#
+#    MODEL_IMAGE_BUILD:
+#      Set to 'when-changed' (default) or 'always'. Default behavior is to
+#      exit without building if the docker image BASE_IMAGE_NAME:BASE_IMAGE_TAG
+#      is found in the local docker image cache.
+#
+#    WDT_DOMAIN_TYPE
+#      'WLS' (default), 'RestrictedJRF', or 'JRF'.
+#
+#    BASE_IMAGE_NAME, BASE_IMAGE_TAG:
+#      The base image name defaults to 
+#         'container-registry.oracle.com/middleware/weblogic'
+#      for the 'WLS' domain type, and otherwise defaults to 
+#         'container-registry.oracle.com/middleware/fmw-infrastructure'. 
+#      The tag defaults to '12.2.1.4'.
+#
+#    MODEL_IMAGE_NAME, MODEL_IMAGE_TAG:
```

Comment:
Should just be a single argument.  Also, call to WIT below uses variable MODEL_IMAGE, which is not created from these variables in this script.

Reply:
This can change - I agree it's confusing, but I'm following the tradition of the other samples.

Reply:
I think the idea is that customers will commonly want to change the tag independent of the image name...